x-pack/filebeat/docs/inputs/input-aws-s3.asciidoc

```diff
@@ -231,6 +231,7 @@ Specific AWS permissions are required for IAM user to access SQS and S3:
 
 ----
 s3:GetObject
+s3:ListBucket
```

Comment:
I think we should separate the permissions required by usage mode (whether using it with sqs or just s3 alone).

Reply:
fine for me